package.json

```diff
@@ -44,48 +44,26 @@
   "author": "Anton Golub <a.golub@qiwi.com>",
   "license": "MIT",
   "dependencies": {
-    "tslib": "^2.2.0"
+    "tslib": "^2.4.0"
   },
   "devDependencies": {
-    "@semantic-release/changelog": "^5.0.1",
-    "@semantic-release/git": "^9.0.0",
-    "@semantic-release/github": "^7.2.3",
-    "@semantic-release/npm": "^7.1.3",
-    "@types/jest": "^26.0.23",
-    "@types/jest-json-schema": "^2.1.3",
-    "coveralls": "^3.1.0",
-    "dts-generator": "^3.0.0",
-    "eslint-plugin-typescript": "^0.14.0",
+    "@qiwi/libdefkit": "^4.0.8",
+    "@qiwi/npm-run-all": "^4.1.7",
+    "@types/jest": "^29.2.0",
+    "@types/jest-json-schema": "^6.1.1",
+    "coveralls": "^3.1.1",
+    "eslint": "^8.26.0",
+    "eslint-config-prettier": "^8.5.0",
+    "eslint-config-qiwi": "^1.17.8",
     "esm": "^3.2.25",
-    "flowgen": "1.14.1",
-    "jest": "^27.0.4",
+    "jest": "^29.2.2",
     "lodash": "^4.17.21",
-    "replace-in-file": "^6.2.0",
-    "semantic-release": "^17.4.3",
-    "ts-jest": "27.0.4",
-    "tslint": "^6.1.3",
-    "tslint-config-qiwi": "^1.7.1",
-    "tslint-config-standard": "^9.0.0",
-    "typedoc": "^0.20.36",
-    "typescript": "^4.3.2",
-    "typescript-eslint-parser": "^22.0.0"
-  },
-  "release": {
-    "branch": "master",
-    "verifyConditions": [
-      "@semantic-release/changelog",
-      "@semantic-release/npm",
-      "@semantic-release/git"
-    ],
-    "prepare": [
-      "@semantic-release/changelog",
-      "@semantic-release/npm",
-      "@semantic-release/git"
-    ],
-    "publish": [
-      "@semantic-release/npm",
-      "@semantic-release/github"
-    ]
+    "microbundle": "^0.15.1",
+    "replace-in-file": "^6.3.5",
```

Comment:
Not sure if this is a required dep now. Let's add [depcheck](https://github.com/qiwi/blank-ts-repo/blob/master/package.json#L27) to verify this point.